semantics/language/common/typing/misc.k

```diff
@@ -331,6 +331,7 @@ module C-TYPING-MISC
      rule getReadFrom'(K:K, S:Set) => getReadFrom'(.K, S)
           requires getKLabel(K) =/=KLabel 'readFrom
                andBool (K =/=K .K)
+     rule getReadFrom'(.K, .Set) => NullPointer
```

Comment:
this does not seem correct to me... I believe the motivation is supposed to be that this function is only called in contexts where hasReadFrom has already evaluated to true. Maybe we should create some kind of special bottom value if you need the function to be total, but NullPointer is a valid SymLoc, which is not where this particular value was actually read from.